src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/subcommand/AboutSubcommand.java

```diff
@@ -38,12 +45,41 @@ public Integer call() throws IOException, SQLException {
         "This tool allows to extract meta information from a data warehouse that allows");
     System.out.println("to make an assessment for migration.");
     System.out.println();
-    try (InputStreamReader reader =
-        new InputStreamReader(this.getClass().getResource("/NOTICE.txt").openStream())) {
-      for (String s : CharStreams.readLines(reader)) {
-        System.out.println(s);
+    System.out.println("Data Warehouse Assessment Extraction Client");
```

Comment:
Is it expected to print this line twice? (Line 42)

Reply:
Thank you, I didn't notice this. I guess once is enough. Fixed.